src/CONST.js

```diff
@@ -991,7 +991,7 @@ const CONST = {
         CODE_2FA: /^\d{6}$/,
         ATTACHMENT_ID: /chat-attachments\/(\d+)/,
         HAS_COLON_ONLY_AT_THE_BEGINNING: /^:[^:]+$/,
-        NEW_LINE_OR_WHITE_SPACE: /[\n\s]/g,
+        NEW_LINE_OR_WHITE_SPACE_OR_EMOJI: /([\uD800-\uDBFF][\uDC00-\uDFFF]|[\n\s])/g,
```

Comment:
Hey @dukenv0307 I was curious in the proposal too about this, **how did you come up with this regex?** 
I tried searching the web real quick here and didn't find a standard regex for checking for emojis